code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -73,6 +82,9 @@
 /obj/item/weapon/spellbook/proc/get_available_artifacts()
 	return available_artifacts
 
+/obj/item/weapon/spellbook/proc/get_available_potions()
+	return available_potions.Copy()
```

Comment:
Why are you copying this?

Reply:
Only because that's what `get_available_spells()` does.

Reply:
Not an excuse.

Reply:
I wasn't trying to make an excuse, I just figured there was a reason for it. I'll remove it if not.